spec/models/user_spec.rb

```diff
@@ -328,6 +328,20 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
       end
     end
 
+    describe "#youtube_url" do
+      it "accepts valid https youtube url", :aggregate_failures do
+        %w[thepracticaldev thepracticaldev/ the.practical.dev].each do |username|
+          user.youtube_url = "https://youtube.com/#{username}"
+          expect(user).to be_valid
+        end
+      end
+
+      it "does not accept invalid facebook url" do
```

Comment:
Probably want to update the description of this spec so there is no confusion 😉

Reply:
Awesome. Thanks for taking your time to review the PR.